### PR TITLE
New news model, serializers, views Kucher

### PR DESCRIPTION
**Pull request by MetalMari**

### CODE REVIEW

File: codeArena/codeArena/settings.py [hunk 1]
```diff
@@ -9,9 +9,12 @@
 
 from pathlib import Path
 from decouple import config
+import os
```
Comment (vvelizariy) [inline codeArena/codeArena/settings.py]: Please format imports according PEP8

File: codeArena/codeArena/urls.py [hunk 2]
```diff
@@ -15,8 +15,18 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
```
Comment (vvelizariy) [inline codeArena/codeArena/urls.py:17]: Keep alphabetically ordered imports

File: codeArena/news/forms.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from django import forms
+from .models import News
+
+
+class NewsForm(forms.ModelForm):
+    class Meta:
+        model = News
+        fields = ('title', 'content', 'src')
```
Comment (vvelizariy) [inline codeArena/news/forms.py]: Dont forget about empty new lines athe end of file

File: codeArena/news/templates/index.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<form method="post" enctype="multipart/form-data">
```
Comment (vvelizariy) [inline codeArena/news/templates/index.html:1]: You should upload files using React, instead of Django templates

File: codeArena/news/urls.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from .views import NewsView, GetNewsDetailView, CreateNewsView
+app_name = "news"
```
Comment (vvelizariy) [inline codeArena/news/urls.py]: 2 empty lines after imports, please check PEP8 styleguide

File: codeArena/news/views.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.views import APIView
```
Comment (vvelizariy) [inline codeArena/news/views.py:3]: Please follow PEP8 imports styleguide

File: codeArena/user/migrations/0001_initial.py [hunk 7]
```diff
@@ -1,4 +1,5 @@
-# Generated by Django 3.2.5 on 2021-09-06 17:38
+
+# Generated by Django 3.2.5 on 2021-09-03 00:51
```
Comment (vvelizariy) [inline codeArena/user/migrations/0001_initial.py]: Remove this line as such info is useless

File: codeArena/news/templates/index.html (hunk 4 above)
Comment (MetalMari) [reply codeArena/news/templates/index.html:1]: Yes, but we did not provide a page and functionality for creating news in our project

File: codeArena/codeArena/urls.py (hunk 2 above)
Comment (MetalMari) [reply codeArena/codeArena/urls.py:17]: Done

File: codeArena/codeArena/settings.py (hunk 1 above)
Comment (MetalMari) [reply codeArena/codeArena/settings.py]: Done

**Review by OleksandrHavron — APPROVED** [review]

File: codeArena/news/templates/index.html (hunk 4 above)
Comment (vvelizariy) [reply codeArena/news/templates/index.html:1]: Ok, you may keep it temporarily. But do not forget to remove templates, when React component for this functionality will be implemented.

**Review by Maksson — APPROVED** [review]

**Review by dmytrosmetanin — APPROVED** [review]
approved

**Pull request merged**